ratatui-widgets/src/table.rs

```diff
@@ -824,6 +824,48 @@ impl Table<'_> {
         }
     }
 
+    /// Return the space that a Cell should occupy, taking into account its colspans
+    ///
+    /// Returns `None` when there are no more columns for the Cell to occupy; in this case,
+    /// rendering of the rest of the row can safely be skipped.
+    /// Otherwise, returns a pair `Some((x, width))`, representing the start x-coordinate and width
+    /// of the Cell.
```

Comment:
for better readability:

```suggestion
    /// Return the space that a Cell should occupy, taking into account its colspans.
    ///
    /// Returns `None` when there are no more columns for the Cell to occupy; in this case,
    /// rendering of the rest of the row can safely be skipped.
    ///
    /// Otherwise, returns a pair `Some((x, width))`, representing the start x-coordinate and width
    /// of the Cell.
```